documentation/Query-Syntax.md

```diff
@@ -127,22 +130,75 @@ Used to extract values from current state, for example, from `user`, `organisati
 Performs queries to a connected PostGres database and returns the result in a format specified by `type`.
 
 - Input:
-  - First child node contains a **string** representing the parameterized SQL query (i.e. $1, $2 substitution)
-  - Remaining nodes return the values (**strings**, **numbers**, **arrays**) required for the above query substitution.
-- Output: Either **array** (all values flattened into one array), **string** (all results concatenated with spaces), or a single **number**, depending on the type specification. If not specified, it returns the default `node-postgres` format (an array of objects, with fields of each object being the database column names).
+  - 1st child node returns a **string** containing the parameterized SQL query (i.e. `$1`, `$2` substitution)
```

Comment:
```suggestion
  - 1st child node's returns a **string** containing the parameterized SQL query (i.e. `$1`, `$2` substitution)
```
This is a new English one for me, but noticed you have that on the following

Reply:
Oh right, it's the other one that's bad English, will change that -- line 162. Thanks :)